src/inspect_ai/util/_sandbox/environment.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Required for referencing `SandboxEnvironmentConfigType` from the `SandboxEnvironment` class methods.